### DISCUSSION TITLE
No ViewManager found for class RNSScreenStackHeaderConfig

### DISCUSSION THREAD

**Discussion by GANESHSHARMA1** [software-mansion/react-native-svg#2468] [Q&A]
# Can anyone help

![image](https://github.com/user-attachments/assets/016054c5-a939-4da8-b764-77dc6d2347cd)

## This is My Code

![image](https://github.com/user-attachments/assets/a54efe15-1675-49b8-82c9-61f4aadb2987)

**Comment by bohdanprog** [comment]
Without any repro example, it is hard to help you.
unfortunately,  I believe it's not related to react-native-svg.
